sessions/biases-in-delay-distributions.qmd

```diff
@@ -84,8 +84,7 @@ head(df)
 
 # Dates, not days: censoring
 
-Data on health outcomes are usually not recorded in the way that we have used so far in this session: as a numeric time since a given start date.
-Instead, we usually deal with *dates*.
+So far, the data we've been using has represented time as a *continuous* variable: a numeric time since a given starting point. Data on health outcomes are usually not recorded in this way. Instead, we usually deal with *dates*: a discrete interval. We refer to this restriction of the data as *censoring*.
```

Comment:
```suggestion
So far, the data we've been using has represented time as a *continuous* variable: a numeric time since a given starting point.
Data on health outcomes are usually not recorded in this way.
Instead, we usually deal with *dates*: a discrete interval.
We refer to this restriction of the data, where we only know that an event has occurred within a given time period (e.g. a day) but not when exactly within that period, as *censoring*.
```